easybuild/easyconfigs/b/BEEF/BEEF-0.1.1-intel-2015a.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'BEEF'
+version = '0.1.1'
+
+homepage = 'http://suncat.stanford.edu/facility/software/functional'
+description = """BEEF is a library-based implementation of the Bayesian
+Error Estimation Functional, suitable for linking against by Fortran-
+or C-based DFT codes. A description of BEEF can be found at
+http://dx.doi.org/10.1103/PhysRevB.85.235149."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'openmp': False, 'usempi': False}
+
+source_urls = ['https://confluence.slac.stanford.edu/download/attachments/146704476']
+sources = "libbeef-{0}.tar.gz".format(version)
```

Comment:
this should be a list, and please use `%(version)s` (avoid using Python code when you can in easyconfigs)

```python
sources = ['libbeef-%(version)s.tar.gz']
```